src/metrics/instance_metric_definitions.go

```diff
@@ -172,3 +159,19 @@ type waitTimeModel struct {
 	WaitTime  *int64  `db:"wait_time"`
 	WaitCount *int64  `db:"waiting_tasks_count"`
 }
+
+var diskMetricInBytesDefination = []*QueryDefinition{
```

Comment:
the query was part of "instanceDefinitions" was there a toggle for those already?
Changing "group" does it imply anything?

Reply:
There was no toggle already, all queries part of `instanceDefinitions` run so to make diskbytes query option I took it out and when it's true we add it back to before processing.

IMO There are 2 ways of doing it 
Add the query when the flag is set true or Remove the query when it's set to false.